corehq/util/metrics/datadog.py

```diff
@@ -12,12 +13,17 @@
 datadog_logger = logging.getLogger('datadog')
 
 
+def _sanitize_value(tag_value: str):
+    # valid characters for tag values from https://docs.datadoghq.com/getting_started/tagging/#defining-tags
+    return re.sub(r'[^a-zA-Z0-9_:./-]+', '_', tag_value.lower())
```

Comment:
Is this going to change any existing datadog values? I guess I'm thinking specifically about the `lower()`.

Reply:
Hm, there is no real reason to lower, so seems like I should just get rid of that. Not sure what I was thinking with that one.

Reply:
Oh, I remember. Datadog does this anyway https://docs.datadoghq.com/getting_started/tagging/#defining-tags. So we can remove it or leave it, but it will have no effect

Reply:
I see. So having writing it out here just makes what it already does more explicit for someone who looks at this to try to answer what transformation is done. Makes sense.